hibernate-jfr/src/test/java/org/hibernate/event/jfr/JdbcPreparedStatementEventTests.java

```diff
@@ -85,6 +88,53 @@ public void testJdbcPreparedStatementEventNoFired(SessionFactoryScope scope) {
 		assertThat( events ).hasSize( 0 );
 	}
 
+	@Test
+	@RequiresDialect(H2Dialect.class)
+	@EnableEvent(JdbcPreparedStatementCreationEvent.NAME)
+	@EnableEvent(JdbcPreparedStatementExecutionEvent.NAME)
+	public void testJdbcPreparedStatementEventStoredProcedure(SessionFactoryScope scope) {
+		scope.inTransaction(
+				session -> {
+					ProcedureCall call = session.createStoredProcedureCall("DB_OBJECT_SQL", String.class)
```

Comment:
I just needed some built in procedure I can call http://www.h2database.com/html/functions.html#db_object_sql